src/basic/capability-util.c

```diff
@@ -161,28 +161,21 @@ int capability_ambient_set_apply(uint64_t set, bool also_inherit) {
         return 0;
 }
 
-int capability_bounding_set_drop(uint64_t keep, bool right_now) {
-        _cleanup_cap_free_ cap_t before_cap = NULL, after_cap = NULL;
+int capability_gain_cap_setpcap(cap_t *before_caps) {
```

Comment:
we usually prefix return parameters like this one with `ret_` in the name: i.e. `ret_before_caps`